pkgs/development/ocaml-modules/ocp-ocamlres/default.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ ocaml findlib ];
-  buildInputs = [ astring pprint ];
+  propagatedBuildInputs = [ astring pprint ];
```

Comment:
It might be worth moving the `bin/` directory to a separate output then.

Reply:
Not sure how to do this :thinking:

Reply:
It actually looks like this change was wrong, I'll just revert it